tests/test_api.py

```diff
@@ -278,6 +287,15 @@ def test_get_campaign_camelcase(
             "Main Data Provider": "4Subsea",
             "Start Date": "2017-04-08T00:00:00+00:00",
             "End Date": "2017-05-13T00:00:00+00:00",
+            "GeoTrack Location": "10#21",
```

Comment:
I don't know why it needs all these, I assume they have been added to the API without the package having been built. Perhaps we should run these tests when we deploy 4insight, or at least at some intervals to stay up to date on changes.